service/controller/appcatalog/v1/key/key.go

```diff
@@ -0,0 +1,37 @@
+package key
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	versionBundleAnnotation = "giantswarm.io/version-bundle"
```

Comment:
So this should be actually be `app-operator.giantswarm.io/version` according to the latest spec.

Reply:
If that is a subject to change then this is perfectly fine.